app/src/main/java/com/github/featuredetectandroid/utils/PhotoAnalyzer.kt

```diff
@@ -42,6 +44,17 @@ class PhotoAnalyzer(private val imageViewModel: GrayscaleViewModel) : ImageAnaly
             width = height.also { height = width }
         }
 
+        featureDetector?.width = width
+        featureDetector?.height = height
+        val keypoints = (
+            featureDetector
+                ?.detect(luminanceArrayToRGB(oriented)) ?: Pair(
+                emptyList(),
+                emptyList()
+            )
+            )
+            .first
```

Comment:
Can you format it more compact somehow? Maybe like
```kotlin
val keypoints =
    (featureDetector?.detect(luminanceArrayToRGB(oriented))
        ?: Pair(emptyList(), emptyList())).first
```

Reply:
temp fix